contents/flood_fill/code/java/flood_fill.java

```diff
@@ -0,0 +1,96 @@
+package code.java;
+import java.util.*;
+public class flood_fill {
+        static class Point{
+            int x;
+            int y;
+            public Point(int x,int y){
+                this.x = x;
+                this.y = y;
+            }
+        }
+        public static boolean inbound(int[][] grid, Point p){
+            int row = grid.length, col = grid[0].length;
+            return (p.x < 0 || p.x >= row || p.y < 0 || p.y >= col);
```

Comment:
Sorry I mislead you earlier, this should be the correct way:
```suggestion
            return !(p.x < 0 || p.x >= row || p.y < 0 || p.y >= col);
```